tests/cli/merge-with-similar-versions.wit.invalid-semver.stdout

```diff
@@ -0,0 +1 @@
+
```

Comment:
Mind adding a special case that skips the output if the output is empty? (avoids adding lots of empty files)